src/tests/suites/StringUtility-Test.ts

```diff
@@ -3,71 +3,71 @@
 // ----------------------------------------------------------------------------
 'use strict';
 
-import * as assert from 'assert';
-
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
 import { StringUtility } from '../../utility/StringUtility';
 
-suite(`StringUtility Test`, () => {
-    test(`compareNoCase`, () => {
-        assert.strictEqual(StringUtility.compareNoCase(`hello`, `hello`), true);
-        assert.strictEqual(StringUtility.compareNoCase(`hello`, `hi`), false);
-        assert.strictEqual(StringUtility.compareNoCase(`hello`, undefined), false);
-        assert.strictEqual(StringUtility.compareNoCase(null, `hi`), false);
-        assert.strictEqual(StringUtility.compareNoCase(undefined, undefined), true);
-        assert.strictEqual(StringUtility.compareNoCase(null, null), true);
-        assert.strictEqual(StringUtility.compareNoCase(undefined, null), false);
-        assert.strictEqual(StringUtility.compareNoCase(`hello`, `HELLO`), true);
+describe(`StringUtility Tests`, () => {
+    it(`compareNoCase`, () => {
+        expect(StringUtility.compareNoCase(`hello`, `hello`)).to.equal(true);
+        expect(StringUtility.compareNoCase(`hello`, `hi`)).to.equal(false);
+        expect(StringUtility.compareNoCase(`hello`, undefined)).to.equal(false);
+        expect(StringUtility.compareNoCase(null, `hi`)).to.equal(false);
+        expect(StringUtility.compareNoCase(undefined, undefined)).to.equal(true);
+        expect(StringUtility.compareNoCase(null, null)).to.equal(true);
+        expect(StringUtility.compareNoCase(undefined, null)).to.equal(false);
```

Comment:
💡🛠️ suggestion only please expect(true).to.be.true; - [link](https://www.chaijs.com/api/bdd/#true)

Reply:
done updated.